src/FormBuilder/resources/js/form-builder/src/containers/HeaderContainer.tsx

```diff
@@ -51,7 +52,9 @@ const HeaderContainer = ({
     const onSave = (formStatus: FormStatus) => {
         setSaving(formStatus);
 
-        dispatch(setFormSettings({formStatus}))
+        dispatch(setFormSettings({formStatus}));
+
+        revertMissingBlocks(blocks);
```

Comment:
Ok, cool - this is how I thought it would work.

Reply:
Pretty interesting right? As a note, my theory for having to do this ourselves is because we're using json and not the html delimited structure that WordPress uses.